ros2run/ros2run/command/run.py

```diff
@@ -22,7 +22,7 @@
 
 
 class RunCommand(CommandExtension):
-    """Various run related sub-commands."""
+    """Run a package specific executable."""
```

Comment:
It looks ok as-is to me. I read it as `Run a "package specific" executable.`  Where the quoted string is an adjective and is optional, i.e. it would read as well (though less specific) as `Run an executable.`.